axon-framework/events/event-versioning.md

```diff
@@ -13,6 +13,13 @@ Upcasters are classes that take one input event of revision `x` and output zero
 > **Note**
 >
 > Perhaps the greatest benefit of upcasting is that it allows you to do non-destructive refactoring. In other words, the complete event history remains intact.
+>
+> **Conversion Notice**
+>
+> Sometimes the event store can contain events in different serialized formats, since differing Serializer implementations where used.
```

Comment:
```suggestion
> Sometimes the event store can contain events in different serialized formats, since differing `Serializer` implementations where used.
```